inst/rmarkdown/templates/xaringan/resources/uio.css

```diff
@@ -0,0 +1,75 @@
+a, a > code {
+  color: #F00;
+  text-decoration: none;
+}
+.footnote {
+  position: absolute;
+  bottom: 3em;
+  padding-right: 4em;
+  font-size: 90%;
+}
+.remark-code-line-highlighted {
+  color: #000;
+  background-color: #ffd6d6;
+}
+
+.inverse {
+  background-color: #000;
+  color: #d6d6d6;
+  text-shadow: 0 0 20px #333;
+}
+.inverse h1, .inverse h2, .inverse h3 {
+  color: #f3f3f3;
+}
+/* Two-column layout */
+.left-column {
+  color: #777;
+  width: 20%;
+  height: 92%;
+  float: left;
+}
+.left-column h2:last-of-type, .left-column h3:last-child {
+  color: #000;
+}
+.right-column {
+  width: 75%;
+  float: right;
+  padding-top: 1em;
+}
+.pull-left {
+  float: left;
+  width: 47%;
+}
+.pull-right {
+  float: right;
+  width: 47%;
+}
+.pull-right ~ * {
+  clear: both;
+}
+img, video, iframe {
+  max-width: 100%;
+}
+blockquote {
+  border-left: solid 5px lightgray;
+  padding-left: 1em;
+}
+.remark-slide table {
+  margin: auto;
+  border-top: 1px solid #666;
+  border-bottom: 1px solid #666;
+}
+.remark-slide table thead th { border-bottom: 1px solid #ddd; }
+th, td { padding: 5px; }
+.remark-slide thead, .remark-slide tfoot, .remark-slide tr:nth-child(even) { background: #eee }
+
+@page { margin: 0; }
+@media print {
+  .remark-slide-scaler {
+    width: 100% !important;
+    height: 100% !important;
+    transform: scale(1) !important;
+    top: 0 !important;
+    left: 0 !important;
+  }
+}
```

Comment:
If this CSS file is largely copied from `default.css`, I'd only write the CSS rules that are unique to your theme in this file, and users can use your CSS to override certain rules in `default.css`, e.g.,

```
  xaringan::moon_reader:
    css: [default, uio-fonts, uio]
```

That way, you don't need the maintain the `75 - N` lines of CSS code when you only need the `N` lines of code.

More on the overriding mechanism:

- https://slides.yihui.org/xaringan#37
- https://slides.yihui.org/xaringan#39

Reply:
Thank you for the tip, I'm definitely in favor of a leaner code. I've just come back from vacation, I'll work on this ASAP.

Reply:
Addressed on https://github.com/yihui/xaringan/pull/270/commits/719f7b6e9dd7c00716281246c3284e0dfa17f5de.